docs/reference/mapping/params/analyzer.asciidoc

```diff
@@ -1,81 +1,17 @@
 [[analyzer]]
 === `analyzer`
 
-The values of <<mapping-index,`analyzed`>> string fields are passed through an
-<<analysis,analyzer>> to convert the string into a stream of _tokens_ or
-_terms_.  For instance, the string `"The quick Brown Foxes."` may, depending
-on which analyzer is used,  be analyzed to the tokens: `quick`, `brown`,
-`fox`.  These are the actual terms that are indexed for the field, which makes
-it possible to search efficiently for individual words _within_  big blobs of
-text.
+[IMPORTANT]
+====
+Only <<text,`text`>> fields support the `analyzer` mapping parameter.
+====
 
-This analysis process needs to happen not just at index time, but also at
-query time: the query string needs to be passed through the same (or a
-similar) analyzer so that the terms that it tries to find are in the same
-format as those that exist in the index.
-
-Elasticsearch ships with a number of <<analysis-analyzers,pre-defined analyzers>>,
-which can be used without further configuration.  It also ships with many
-<<analysis-charfilters,character filters>>, <<analysis-tokenizers,tokenizers>>,
-and <<analysis-tokenfilters>> which can be combined to configure
-custom analyzers per index.
```

Comment:
Covered in [Anatomy of an analyzer][0].

[0]:https://www.elastic.co/guide/en/elasticsearch/reference/master/analyzer-anatomy.html

Reply:
Same here, maybe this can be linked in some way?

Reply:
I added a link with 5460b11c802.